cumulusci/tasks/metadata/metadata_map.yml

```diff
@@ -108,6 +108,9 @@ layouts:
 letterhead:
     - type: Letterhead
       class: MetadataFilenameParser
+lwc:
+    - type: LightningComponentBundle
+      class: LightningComponentBundleParser
```

Comment:
Should be `BundleParser` now.

Reply:
Great catch.  I thought I changed this when (re-)defining `BundleParser`

Reply:
Changes pushed